src/library_wasmfs.js

```diff
@@ -18,6 +32,14 @@ FS.init();
 FS.createPreloadedFile = FS_createPreloadedFile;
 `,
   $FS__deps: [
+    '$MEMFS',
+    '$ICASEFS',
+#if LibraryManager.has('library_nodefs.js')
+    '$NODEFS',
+#endif
+    '$OPFS',
+    '$JSFILEFS',
+    '$FETCHFS',
```

Comment:
I agree with the `#ifdef FORCE_FILESYSTEM`. Since `library_wasmfs_opfs.js` is included for every WasmFS compilation, does it still make sense to use `LibraryManager` to check if `library_wasmfs_opfs.js` is present?

Reply:
> Since library_wasmfs_opfs.js is included for every WasmFS compilation, does it still make sense to use LibraryManager to check if library_wasmfs_opfs.js is present?

Correct, that doesn't make sense.

I think the idea we were discussing we met is that we'd check for a new file, `library_opfs.js`(to mirror `library_nodefs.js` etc. Those files are conditionally included, so checking for them makes sense. (Whereas the `_wasmfs_` ones are always included since non-JS code might use them.)

Reply:
Got it. Should there be a check and/or warning if the user tries to link `-lopfs.js` when using the legacy API? If so, would that go somewhere in `emcc.py`?

Reply:
Good idea! Yeah, this can help users. Maybe `library_opfs.js` can end in
```js
if (!WASMFS) {
  error("using -lopfs.js requires using WasmFS (-sWASMFS)");
}
```